src/modules/cursor.c

```diff
@@ -190,6 +215,12 @@ void ffPrintCursor(FFinstance* instance)
         return;
     }
 
+    if(ffStrbufStartsWithIgnCaseS(&wmde->dePrettyName, "LXQt"))
+    {
+        printCursorLXQt(instance);
+        return;
+    }
```

Comment:
Very similar to plasma code, maybe it is possible to write a single function which takes the path to the config file and the default values as arguments and use it for both?

Reply:
I was just thinking that...

Reply:
If it should fall back to GTK you have two options: 
1) call `printCursorGTK` manually
2) return a bool value, for example like the env code

I would go for the second.

Reply:
If it doesn't has defaults and you want to write a single method, just pass `NULL` as default value. `ffStrbufAppendS` doesn't do anything if str is NULL. Maybe falling back to GTK makes sense for plasma too.

Reply:
Possible declaration of the function as a tip:
```c
bool printCursorFromConfigFile(FFinstance* instance, const char* relativeFilePath, const char* themeStart, const char* themeDefault, const char* sizeStart, const char* sizeDefault)
```

Reply:
XCURSOR_THEME and XCURSOR_SIZE is not set on my system, so the env route seems unlikely.

Reply:
Yea this was just meant to be an example for how to handle functions which return bool

Reply:
Ah. I should read better. :grin:

Reply:
Like this to be exactly:
```c
if(ffStrbufIgnCaseCompS(&wmde->dePrettyName, "KDE Plasma") == 0 && printCursorFromConfigFile(instance, "kcminputrc", "cursorTheme =", "Breeze", "cursorSize =", "24")
    return;
```

Reply:
Defaults to [Adwaita](https://wiki.manjaro.org/index.php/LXQt#Change_Mouse_Cursor)